results/32-LV-viper-correlation-of-correlations.Rmd

```diff
@@ -0,0 +1,314 @@
+---
+title: "31 - Check intercorrelation of LV-associated viper proteins"
+author: "Robert Allaway"
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    code_folding: hide
+    toc: true
+    toc_float: true
+---
+
+```{r global_options, include=FALSE}
+knitr::opts_chunk$set(warning=FALSE, message=FALSE)
+```
+
+Suggested by Jineta - let's look at correlation of correlations with regards to the viper proteins associated with each LV. 
+
+# Import packages
+
+First, import packages to process and plot the data. 
+
+```{r message=FALSE, warning=FALSE}
+library(tidyverse)
+library(synapser)
+set.seed(98121)
+synLogin()
+
+viper_res <- synTableQuery('SELECT * FROM syn21260871 where numSamps = 77')$filepath %>% 
+  readr::read_csv() %>%
+  mutate(latent_var_trunc = stringr::str_trunc(latent_var, 15)) 
+
```

Comment:
This is fantastic! I think all the downstream analysis is prefect  - my only concern is this query: I think using the LV/metaviper correlations to produce the clusters makes things more complicated then they need to be. I'm experimenting with some sort of clustering of the viper proteins to see if we get the same 6 clusters, and if we do I'll file a new PR>

Reply:
Interesting - I think that the question for these two different clustering analyses is different but let me know what you think: 

For the analysis in this PR, the question is: Regarding the landscape of proteins/targets predicted to be expressed for each of the latent variables, is there any concordance across the latent variables? 

For the analysis you are trying the question is: Is the expression of certain viper proteins correlated with each other?

Is this correct? I think these are slightly different questions. I think if we want to connect latent variables to druggable targets, the first analysis is what we'd need, but if we don't care so much about that, the second is what we'd want to do.